

<div ui-view layout="column">

    <div class="container">

    <br>
    <img src="ms-projects/events/img/plantseed2016-header.png" style="width: 100%; padding-right: 10px;">

    <br>

    <a class="pull-right" href="http://bioseed.mcs.anl.gov/~seaver/Files/PlantSEED_Workshop_2016/PlantSEED_2016_Workshop_Brochure.pdf" target="_blank">
        Download
        <i class="icon-file-download text-muted"></i>        
    </a>    
    
    <h2 class="text-center">2016 PlantSEED Metabolic Modeling Workshop</h2>

    A two-day workshop on plant genome annotation, metabolic modeling and reconstruction 
    comprising formal lectures and hands-on instruction, will be held at 
    <b>Northwestern University, Evanston, IL from the 4th to the 5th of August, 2016</b>.
    The workshop content is of interest to computational scientists associated to agriculture 
    and biology departments as well as plant scientists interested in metabolic modeling 
    including plant breeders, geneticists, physiologists and biochemists. <b>The workshop is free of charge.</b><br>
    <br>

    <div class="important-note">
        <i class="material-icons pull-left" style="padding-right: 10px;">error_outline</i>
        Please ensure you have RAST account (<a href="http://rast.nmpdr.org/?page=Register" target="_blank">here</a>) 
        and that you can log into the ModelSEED website (<a href="/">here</a>) before the workshop. 
    </div>

    <h3 class="text-center">Program</h3>

    <h3>Workshop Aims</h3>
        <indent>
        - Be familiar with the capabilities of the PlantSEED database and its suite of comparative genomics tools.<br>
        - Be equipped for use of the PlantSEED and related platforms like STRING to annotate genes and use
        subsystems to predict functions for genes.<br>
        - Be able to use advanced PlantSEED functions that increase the efficiency of GENE ANNOTATION and
        allow the user to focus on particular genes and genomes.<br>
        - Generate metabolic reconstruction models by mapping between annotations and reactions.<br>
        - Perform GAP FILLING and connect the curation of gapfilling results with existing external tools (e.g.
        KEGG, PlantCyc).<br>
        - Do FLUX BALANCE ANALYSIS and use modeling to analyze plants metabolism. -Give examples of problem solutions in sandbox models
        </indent>


    <h3>Related Information</h3>

    For more background please read the following manuscripts:<br>
    <br>
        <indent>
        Orth JD, Thiele I, Palsson BØ (2010) What is flux balance analysis? Nat Biotechnol. 28(3):245-8.
        doi: 10.1038/nbt.1614.<br>
        <span class="ms-color-error">[PMID:20212490]</span><br>

        Oberhardt MA, Palsson BØ, Papin JA (2009) Applications of genome-scale metabolic reconstructions. Mol Syst Biol. 5:320.
        doi: 10.1038/msb.2009.77.<br>
        <span class="ms-color-error">[PMID:19888215]</span><br>

        Seaver SM, Gerdes S, Frelin O, Lerma-Ortiz C, et al (2014) High-throughput comparison, functional annotation, and metabolic modeling of plant genomes using the PlantSEED resource. Proc Natl Acad Sci U S A. 111: 9645-50.
        doi: 10.1073/pnas.1401329111.<br>
        <span class="ms-color-error">[PMID: 24927599]</span>
        </indent>


    <h3>Team of Expert Instructors</h3>

    Dr. Christopher Henry and Dr. Samuel Seaver (Argonne National Laboratory and the Computation Institute of the University of Chicago)
    Dr. Donald McCarty (Horticultural Sciences Department, University of Florida)<br>


    <h3>Logistics</h3>
    The workshop takes two full days (9 am to 4 pm with an hour break for lunch).
    It will combine formal instruction and individual hands-on tuition.
    Participants should plan to attend for the whole two days. Lectures and instruction are interspersed in a balanced mix as follows:<br>
    <br>

    <h4><b>Day 1:</b> Annotation and modeling of plant genomes in PlantSEED</h4>

    Welcoming opening: Team introduction and log-in technical assistance for participants<br>
    <br>

    <indent>
        A. Introduction to PlantSEED<br>
        <indent>
            a. What is PlantSEED?<br>
            b. What public genomes and datasets are available?<br>
            c. How was the data derived?<br>
            d. What tools are available?<br>
            e. How is user data managed and shared?<br>
        </indent>
        B. Annotation of transcripts (Learn about signature k-mers and FIGFams)<br>
        <indent>
            a. Introduction and tour of PlantSEED subsystems and reference genomes<br>
            b. Overview of PlantSEED signature k-mers and FIGFams<br>
            c. Tools for submission of sequences for annotation and viewing results<br>
        </indent>
        C. View and Comparison of PlantSEED annotations<br>
        <indent>
            a. Tour of PlantSEED genome viewer, including annotation overview and genome feature pages<br>
            b. Tools for comparison of PlantSEED annotation between two genomes/transcripts<br>
            c. Upload, annotation, and comparison of sequences from multiple fasta files<br>
        </indent>

        D. Introduction to metabolic modeling<br>
        <indent>
            a. What is a metabolic model?<br>
            b. What is flux balance analysis?<br>
            c. How are models useful?<br>
            d. Overview of how annotation maps to reactions<br>
        </indent>

        E. Interactive Walkthrough of PlantSEED using Sandbox Model<br>
        <indent>
            a. Users will construct own copy of a sandbox model<br>
            b. Users use the PlantSEED interface to explore the sandbox model<br>
            c. Users will run flux balance analysis on sandbox model<br>
            d. Users will run gapfilling on sandbox model<br>
            e. Users will edit the sandbox model and associated genome<br>
        </indent>
    </indent>

    <h4><b>Day 2:</b> Application of models to understand and discover plant biology</h4>

    <indent>
        A. Demonstration of model applications with sandbox model<br>
            <indent>
            a. Predicting essential genes and reactions<br>
            b. Finding a missing or erroneous annotation<br>
            c. Identifying the pathways responsible for producing a particular compound<br>
            d. Simulating transcriptome data to predict flux<br>
            e. Integrating plant and microbial models<br>
            </indent>

        B. Question and answer session<br>

        C. Application of full genomes, transcriptomes, and models to answer biological questions<br>
            <indent>
                a. Users apply PlantSEED tools to aid their own research<br>
            </indent>
    </indent>


    <h3>Classroom</h3>
    The workshop will be held in room B211 in the Technological Institute at Northwestern University (2145 Sheridan Rd, Evanston, IL 60208) which has the necessary technical support (projectors, extensions, power outlets near desks).
    Participants need to bring their personal Windows or Mac OS X computers. <b>Participants must have Chrome (prefered) or firefox installed on their computers, and fully up-to-date.</b>
    <br>

    <h3>REQUIREMENTS</h3>
    To benefit from the course, applicants need:<br>
    <br>
    - To have some familiarity with the NCBI, detecting sequence homology via alignment phylogenetic
    analysis.<br>
    - To be acquainted with the biochemistry available in KEGG and Plant Metabolic Networks and phylogenetic gene trees in Gramene/ Ensembl and Phytozome.<br>
    - <b class="ms-color-error">To maximize the relevance of the hands-on instruction and increase motivation, registered
    participants will be required to load their own set of gene or transcript sequences from a plant
    of choice in the workshop registration webpage, three weeks before the workshop.</b><br>
    - Some lectures may be recorded and later posted to our website.<br>


    <h3>APPLICATION PROCEDURE, DEADLINES AND PARTICIPANTS SELECTION</h3>
    - The workshop can only accommodate up to 20 participants coming from Northwestern and Chicago Universities as well as other colleges and universities. Faculty as well as postdocs and graduate students may participate.<br>
    - Candidates wishing to participate in our workshop should fill the “PlantSEED Workshop Application Form” (PSWA form) accompanying this brochure, no later than the first of June and send it with to clerma@ufl.edu<br>
    - Applicants will be selected in a first-arrive, first-served basis as well as their qualifications descriptions in order not to exceed 20 subscriptions. Accepted participants will receive an acceptance email as well as a set of instructions about registration, choosing a plant genome and loading it in PlantSEED. Also a detailed workshop/travel general guide will be supplied.<br>
    - <b class="ms-color-error">If you work outside the Chicago area, it is very important to apply as soon as possible, so you can make your hotel and airplane reservations in a timely manner, that is, as soon as you are approved to register. Remember the workshop is free of charge.</b><br>



    <h3>TRAVEL EXPENSES REFUNDS</h3>
    - Up to $1,000 for travel, food and lodging expenses will be awarded only to <b>three faculty members</b> who work at a minority serving institution and are chosen by the panel. To qualify for funding, applicants need to fulfill the following requirements:
    <br>
        <indent>
        1) To be an American citizen or Permanent Resident of the USA.<br>
        2) To be a faculty member of a US College or University that is registered as Minority Serving Institution. If you are unsure about your college being this kind of institution go to: https://www2.ed.gov/about/offices/list/ocr/edlite-minorityinst.html<br>
        </indent>
    <br>
    - These refunds will be paid through the University of Florida, it takes some time to process the paperwork and it has to be completed at least <b class="ms-color-error">a month before the workshop starts</b>. Hence the applications have to be processed with a great deal of anticipation. The information included in the PSWA form will allow our coordination panel to screen the applications to ensure that the three selected faculty of Minority Serving Institutions have the background needed to benefit most from the workshop.<br>
    If you have other questions regarding our workshop please contact Dr. Sam Seaver at: samseaver@gmail.com or Dr Claudia Lerma-Ortiz at: clerma@ufl.edu
    <br><br><br>
    </div>
</div>
