

<div layout="column">
    <md-content layout="column" layout-margin layout-padding>

        <md-tabs md-selected="tabs.tabIndex" md-no-ink md-dynamic-height md-no-bar>


            <md-tab label="Genes">
                <md-tab-body>
                    <ng-include src="'ms-projects/regulons/genes.html'" ng-controller="RegulonsGenes"></ng-include>
                </md-tab-body>
            </md-tab>

            <md-tab label="Regulators">
                <md-tab-body>
                    <ng-include src="'ms-projects/regulons/regulators.html'" ng-controller="RegulonsRegulators"></ng-include>
                </md-tab-body>
            </md-tab>

            <md-tab label="Legend">
                <md-tab-body>

                    <h4>Regulator Descriptions</h4>

                    The character "<b>@</b>" in a column means that the corresponding relation is absent:<br>
                    <p style="margin-left: 15px;">
                    "<b>@</b>" in the column BSU indicates that there is no locustag related to the regulator.<br>
                    "<b>@</b>" in the column metabolite means that the activity of the regulator is not modulated by a metabolite.
                        Consequently, the column metabolite_sign is assigned to "nan" in that case.<br><br>
                    </p>
                    <ul class="list-unstyled">
                        <li>"<b>unk</b>" in the column metabolite means that a metabolite may modulate the activity of the regulator but it has not been identified. The corresponding number and sign is thus assigned to "<b>nan</b>".</li>
                        <li>"<b>nan</b>" in the column metabolite means that the activity of the regulator may be affected by an unknown mechanism. The corresponding number and sign is thus assigned to "<b>nan</b>".</li>
                    </ul>

                    <br>
                    <h4>Regulator Classification</h4>

                    <div layout="row" flex="70">
                    <table class="table table-stripper table-horizontal table-condensed">
                        <thead><thead>
                        <tbody>
                            <tr><td>sigma factor</td><td>Sigma factor</td></tr>
                            <tr><td>sigma factor - ECF type</td><td>Extracytoplasmic function (ECF) sigma factor</td></tr>
                            <tr><td>TF</td><td>Transcription factor </td></tr>
                            <tr><td>TF-TC</td><td>Two-component response regulator </td></tr>
                            <tr><td>TF+M</td><td>Transcription factor conditioned by a metabolite</td></tr>
                            <tr><td>TF+unk</td><td>Transcription factor conditioned by an unknown mechanism/protein/metabolite</td></tr>
                            <tr><td>TF+PP</td><td>Transcription Factor + phosphorylated protein </td></tr>
                            <tr><td>TF+PP+M</td><td>Transcription Factor + phosphorylated protein  + metabolite</td></tr>
                            <tr><td>TF+P</td><td>Transcription Factor  + accessory protein </td></tr>
                            <tr><td>TF+P+M</td><td>Transcription Factor  + accessory protein associated to a metabolite</td></tr>
                            <tr><td>TF+S</td><td>Transcription Factor   + stress (DNA alteration/TF alteration)</td></tr>
                            <tr><td>TF-proteolyse</td><td>Transcription Factor and proteolysis of the TF</td></tr>
                            <tr><td>TF+unstability</td><td>Transcription Factor that is unstable (and thus sensitive to temperature)</td></tr>
                            <tr><td>TF+M+unk</td><td>Transcription factor conditioned by an unknown metabolite</td></tr>
                            <tr><td>TF+ADN-M</td><td>Transcription factor conditioned by a metabolite and by specific  DNA structure</td></tr>
                            <tr><td>P-PTC</td><td>Protein post-transcriptional control</td></tr>
                            <tr><td>P-AT+PTS </td><td>Protein - transcriptional antiterminator conditioned by a PTS phosphorylation</td></tr>
                            <tr><td>P-AT+M</td><td>Protein - transcriptional antiterminator conditioned by a metabolite</td></tr>
                            <tr><td>P</td><td>Accessory protein involved in regulation</td></tr>
                            <tr><td>Riboswitch</td><td>Binding of metabolite to a RNA  </td></tr>
                            <tr><td>RNA switch</td><td>Binding of a tRNA or a protein to a mRNA</td></tr>
                            <tr><td>Rna-BPA</td><td>RNA-binding attenuation protein</td></tr>
                            <tr><td>small regulatory RNA</td><td>small non-coding RNA involved in regulation</td></tr>
                            <tr><td>small RNA - translation</td><td>small non-coding RNA involved in the regulation of mRNA translation</td></tr>
                            <tr><td>RNA-anti antiterminator</td><td></td></tr>
                            <tr><td>anti-sense RNA</td><td>antisense RNA</td></tr>
                            <tr><td>unk</td><td>unknown mechanism of regulation</td></tr>
                            <tr><td>Translation regulation</td><td>Regulation of translation by ribosomes</td></tr>
                            <tr><td>silico-TF+M</td><td>Transcription factor conditioned by a metabolite inferred by <a href="http://regprecise.lbl.gov/RegPrecise/genome.jsp?genome_id=52" target="_blank">RegPrecise</a></td></tr>
                            <tr><td>silico-TF+unk</td><td>Transcription factor conditioned by an unknown mechanism/protein/metabolite inferred by <a href="http://regprecise.lbl.gov/RegPrecise/genome.jsp?genome_id=52" target="_blank">RegPrecise</a></td></tr>
                            <tr><td>silico RNA switch</td><td>Binding of a tRNA or a protein to a mRNA  inferred by <a href="http://regprecise.lbl.gov/RegPrecise/genome.jsp?genome_id=52" target="_blank">RegPrecise</a></td></tr>
                            <tr><td>silico-riboswitch</td><td>Binding of metabolite to a RNA  inferred by <a href="http://regprecise.lbl.gov/RegPrecise/genome.jsp?genome_id=52" target="_blank">RegPrecise</a></td></tr>
                            <tr><td>silico-TF+TC</td><td>Two-component response regulator inferred by <a href="http://regprecise.lbl.gov/RegPrecise/genome.jsp?genome_id=52" target="_blank">RegPrecise</a></td></tr>
                        </tbody>
                    </table>
                </div>
                </md-tab-body>
            </md-tab>


            <md-tab label="Example">
                <md-tab-body>
                    <img src="ms-projects/regulons/img/example.png" style="width: 100%;">
                </md-tab-body>
            </md-tab>

            <md-tab label="Abstract">
                <md-tab-body>
                    <br>
                    <b>Abstract</b><br>
                    <br>
                    <div class="publication">
                        <div class="title">Reconstruction of the Regulatory Network for Bacillus Subtilis and Reconciliation with Gene Expression Data</div>
                        <div class="author">José P. Faria, Ross Overbeek, Ronald C. Taylor, Neal Conrad, Veronika Vonstein, Anne Goelzer, Vincent Fromion, Miguel Rocha, Isabel Rocha, and Christopher S. Henry</div>

                        <div class="abstract">
                            We introduce a manually constructed and curated regulatory network model that describes the current state of knowledge of transcriptional regulation of B. subtilis. The model corresponds to an updated and enlarged version of the regulatory model of central metabolism originally proposed in 2008. We extended the original network to the whole genome by integration of information from DBTBS, a compendium of regulatory data that includes promoters, transcription factors (TFs), binding sites, motifs and regulated operons. Additionally, we consolidated our network with all the information on regulation included in the SporeWeb and Subtiwiki community-curated resources on B. subtilis. Finally, we reconciled our network with data from RegPrecise, which recently released their own less comprehensive reconstruction of the regulatory network for B. subtilis. Our model describes 275 regulators and their target genes, representing 30 different mechanisms of regulation such as TFs, RNA switches, Riboswitches and small regulatory RNAs. Overall, regulatory information is included in the model for approximately 2500 of the ~4200 genes in B. subtilis 168. <br>
                            <br>
                            In an effort to further expand our knowledge of B. subtilis regulation, we reconciled our model with expression data. For this process, we reconstructed the Atomic Regulons (ARs) for B. subtilis, which are the sets of genes that share the same “ON” and “OFF” gene expression profiles across multiple samples of experimental data. We show how atomic regulons for B. subtilis are able to capture many sets of genes corresponding to regulated operons in our manually curated network. Additionally, we demonstrate how atomic regulons can be used to help expand or validate the knowledge of the regulatory networks by looking at highly correlated genes in the ARs for which regulatory information is lacking. During this process, we were also able to infer novel stimuli for hypothetical genes by exploring the genome expression metadata relating to experimental conditions, gaining insights into novel biology.<br>
                            <br>
                        </div>

                        <b>Citation</b><br>
                        <div class="citation">
                            José P. Faria, Ross Overbeek, Ronald C. Taylor, Neal Conrad,
                            Veronika Vonstein, Anne Goelzer, Vincent Fromion, Miguel Rocha,
                            Isabel Rocha, and Christopher S. Henry,
                            “Reconstruction of the regulatory network for Bacillus Subtilis
                            and reconciliation with gene expression data”, Frontiers in Microbiology, 2015.
                            (Manuscript Submitted)
                        </div>
                    </div>
                </md-tab-body>
            </md-tab>
        </md-tabs>
    </md-content>
</div>
